From b131e01aeb71c5d8cf5e4c065fdc4703465f85c1 Mon Sep 17 00:00:00 2001
From: "M.P. Korstanje" <rien.korstanje@gmail.com>
Date: Thu, 23 Jul 2020 15:43:04 +0000
Subject: [PATCH] Fix Gherkin

(cherry picked from commit da99309370c95b20dc358de7c6a610bd348f090b)

Backport only messages.go for messages-go v12.4.0 compatibility. Keep the v13 module path and existing dependency declarations.

Source: https://github.com/cucumber/gherkin-go/commit/da99309370c95b20dc358de7c6a610bd348f090b
---
 messages.go | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/messages.go b/messages.go
index a288300..81aa2a4 100644
--- a/messages.go
+++ b/messages.go
@@ -114,7 +114,9 @@ func (a *parseError) asMessage(uri string) *messages.Envelope {
 			ParseError: &messages.ParseError{
 				Message: a.Error(),
 				Source: &messages.SourceReference{
-					Uri: uri,
+					Reference: &messages.SourceReference_Uri{
+						Uri: uri,
+					},
 					Location: &messages.Location{
 						Line:   uint32(a.loc.Line),
 						Column: uint32(a.loc.Column),
-- 
2.43.0

